sql/modules/Roles.sql

```diff
@@ -434,15 +567,26 @@ SELECT lsmb__grant_perms('contact_edit', 'eca_to_contact', 'ALL');
 SELECT lsmb__grant_perms('contact_edit', 'eca_to_location', 'ALL');
 SELECT lsmb__grant_perms('contact_edit', 'eca_tax', 'ALL');
 
-SELECT lsmb__create_role('contact_delete');
+SELECT lsmb__create_role('contact_delete',
+                         $DOC$
+                         This role allows removal of existing contacts.
+
+                         Combine this role with one or more 'contact_class_*' roles to be able to access
+                         contacts of the specific class. By itself, this role does not provide sufficient rights.
+                         $DOC$
+);
```

Comment:
I suggest a similar wording as the one suggested in `contact_read`.

Also, this role `contact_delete` is not mentioned in the description of the `contact_class_*` roles.

Reply:
I saw in the chat that you actually also need the `contact_read` role for this role to have effect, unless I misunderstood something. You might want to mention this here to (and in `contact_edit` and  `contact_create` if it's true for those as well. A suggestion would be:

>  When paired with one or more `contact_class_*`-role/-s, this role allows removal of existing contacts.
>
> Each `contact_class_<resource>`-role, when paired with `contact_delete`, enables this access for the specific `<resoucre>`. On it's own, the `contact_delete`-role does not provide any rights. Note that you also need `contact_read` for `contact_delete` to be applied as expected.

Reply:
Used different wording, but, yes, this comment is incorporated too.